miasm2/analysis/dse.py

```diff
@@ -215,7 +218,7 @@ def breakpoint(self, jitter):
         self.prepare()
 
     def handle(self, cur_addr):
-        """Handle destination
+        r"""Handle destination
```

Comment:
Is this on purpose?

Reply:
yes:
There is /!\ which makes a dummy escaped char in the comments. this makes autopep8 complain